specification/monitor/resource-manager/Microsoft.Insights/preview/2017-05-01-preview/subscriptionDiagnosticsSettings_API.json

```diff
@@ -325,9 +325,7 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "description": "The subscription id.",
-      "x-ms-parameter-location": "method",
-      "x-ms-skip-url-encoding": true
+      "description": "The Azure subscription Id."
```

Comment:
The autorest.java generator will throw errors for the different values of same parameter.property within the json files under same package tag.